pkg/ebpf/tracer.go

```diff
@@ -177,15 +183,15 @@ func (t *Tracer) expvarStats() {
 			for metric, val := range tracerStats.(map[string]int64) {
 				currVal := &expvar.Int{}
 				currVal.Set(val)
-				probeExpvar.Set(snakeToCapInitialCamel(metric), currVal)
+				tracerExpvar.Set(snakeToCapInitialCamel(metric), currVal)
```

Comment:
We could do:

```go
for name, mp := range stats {
  for metric, val := range mp.(map[string]int64) {
    currVal := &expvar.Int{}
    currVal.Set(val)
    expvarEndpoints[name].Set(snakeToCapInitialCamel(metric), currVal)
  }
}

Reply:
This couldn't really be done, each endpoint is treated slightly differently.